test/inductor/test_binary_folding.py

```diff
@@ -1,4 +1,4 @@
-# Owner(s): ["module: inductor"]
+# Owner(s): ["oncall: cpu inductor"]
```

Comment:
Looks like these tests have also some CUDA testing, so they are not only cpu:
https://github.com/pytorch/pytorch/pull/115167/files#diff-32177bda568814a1368e62144d5b08ad18bb92f5578d9994a5350c516d5e0294R35

Reply:
@jgong5 Thank you!

Reply:
> @jgong5 Thank you! Please proceed as discussed on Slack by decoupling CPU and CUDA tests and assigning `# Owner(s): ["oncall: cpu inductor"]` to cpu tests

Thanks. Will do that in a separate PR.